.eslintrc.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
     '@typescript-eslint/ban-ts-ignore': 'off',
     '@typescript-eslint/camelcase': 'off',
     '@typescript-eslint/no-undef': 'off',
-    '@typescript-eslint/no-var-requires': 'off'
+    '@typescript-eslint/no-var-requires': 'off',
+    '@typescript-eslint/no-use-before-define': ['error', { functions: false }]
```

Comment:
Any reason to remove this rule?

Reply:
I was getting this error when I was calling a function before it was defined. This is common within the Composition API, when structuring a component by feature. Like here: 
![image](https://user-images.githubusercontent.com/51686767/115989694-c53cef00-a5e9-11eb-8869-02b7b1a887ad.png)

Reply:
Why not moving the function before the onMounted? Wouldnt that make the code more clear?

Reply:
Depends how you look at it, from an Options API standpoint I guess it would. But it wouldn't be ordered by feature anymore.

Reply:
I see yes good point. You can leave it like this, i think composable is how we will scope code by feature